shells/chrome/manifest.json

```diff
@@ -18,8 +20,7 @@
 
   "permissions": [
     "http://*/",
-    "https://*/*",
-    "file://*"
```

Comment:
Chrome Web Store doesn't allow it – the extension was automatically rejected. :(